grails-app/services/org/pih/warehouse/core/LocalizationService.groovy

```diff
@@ -64,15 +64,7 @@ class LocalizationService {
      * Gets the current locale or return default locale.
      */
     Locale getCurrentLocale() {
-        // First search for session locale, if none set, then search for user's default locale, otherwise get the default locale
-        def session = RequestContextHolder.currentRequestAttributes().getSession()
-        if (session?.locale) {
-            return session.locale
-        }
-        if (session?.user?.locale) {
-            return session.user.locale
-        }
-        return new Locale(grailsApplication.config.openboxes.locale.defaultLocale ?: "en")
+        return LocaleContextHolder.locale ?: new Locale(grailsApplication.config.openboxes.locale.defaultLocale ?: "en")
```

Comment:
as I wrote in the ticket - I will check it locally, since I want to understand the need of that change and why it was not working and if it didn't cause any regression issues related to crowdin